parachains/runtimes/assets/asset-hub-polkadot/src/lib.rs

```diff
@@ -122,8 +128,8 @@ impl_opaque_keys! {
 
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("statemint"),
-	impl_name: create_runtime_str!("statemint"),
+	spec_name: create_runtime_str!("asset-hub-polkadot"),
+	impl_name: create_runtime_str!("asset-hub-polkadot"),
```

Comment:
@bkchr : Will this affect sync? I'm wondering because the chain spec files have `name` and `id` fields, so I'm not sure if it's possible to change this.

- If we change the chain spec, will it sync from genesis where `spec_name == "statemint"`?
- If we change the spec name, will the node continue to sync after the block at which the name changes?